### PR TITLE
fix `nox -s build` not gathering typeshed libs licenses properly

### PR DESCRIPTION
**Pull request by mjurbanski-reef**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ jobs:
           fetch-depth: 0
       - uses: codespell-project/actions-codespell@2391250ab05295bddd51e36a8c6295edb6343b0e
         with:
-          ignore_words_list: datas
+          ignore_words_list: datas re-use
```
Comment (kkalinowski-reef) [inline .github/workflows/ci.yml:21]: How about excluding the LICENSE file entirely with:
```
with:
  exclude_file: src/foo
```
taken from https://github.com/codespell-project/actions-codespell

File: b2/console_tool.py [hunk 2]
```diff
@@ -3454,6 +3454,8 @@ class License(Command):  # pragma: no cover
             'https://raw.githubusercontent.com/PythonCharmers/python-future/master/LICENSE.txt',
         'pefile':
             'https://raw.githubusercontent.com/erocarrera/pefile/master/LICENSE',
+        'https://github.com/python/typeshed':
```
Comment (kkalinowski-reef) [inline b2/console_tool.py:3457]: Are we using the repository version? Why? Even their original readme says that you don't need to and points to:
```
pip install types-six types-requests
```
instead. Those these libraries provide no licensing information?
https://github.com/python/typeshed#using

**Review by kkalinowski-reef — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (mjurbanski-reef) [reply .github/workflows/ci.yml:21]: it also errored out in other files on the same word

File: b2/console_tool.py (hunk 2 above)
Comment (mjurbanski-reef) [reply b2/console_tool.py:3457]: 1) no, we aren't. This is URL extracted by pip-licenses. In fact this is not even a direct dependency of ours, so we don't specify it anywhere ourselves.
2) piplicense is unable to get LICENSE text for it. It is able to extract just license type (how it does it, idk, but I expect it to be in at least python trove classifiers for this package). It is not enough for our purpose, hence the direct link.

**Pull request merged**